StarLight_Line/GameLib/Wnd/Wnd.cpp

```diff
@@ -0,0 +1,99 @@
+﻿#include <windows.h>
+#include <d3dx9.h>
+
+#include "../Interface/IDX3D/IDX3D.h"
+#include "Wnd.h"
+
+IDX3D* Wnd::m_pIDX3D = nullptr;
+
+Wnd::Wnd(const HINSTANCE hInst, const TCHAR* pAppName, IDX3D* pIDX3D)
+{
+	m_pIDX3D = pIDX3D;
+
+	ZeroMemory(&m_msg, sizeof(MSG));
+
+	Create(hInst, pAppName);
+}
+
+const HWND& Wnd::GetHWND()
+{
+	return m_hWnd;
+}
+
+const MSG& Wnd::GetMSG()
+{
+	return m_msg;
+}
+
+D3DXVECTOR2 Wnd::GetWndSize()
+{
+	return m_WND_SIZE;
+}
+
+BOOL Wnd::ExistsWinMSG()
+{
+	if (PeekMessage(&m_msg, NULL, 0U, 0U, PM_REMOVE))
+	{
+		TranslateMessage(&m_msg);
+		DispatchMessage(&m_msg);
+
+		return FALSE;
```

Comment:
ここに来たときは「郵便受けに手紙が入っている状態」なのでは？
であるならば
```suggestion
		return TRUE;
```
ですよね？

Reply:
そうですねミスです